.github/workflows/build.yml

```diff
@@ -0,0 +1,46 @@
+name: Build
+
+on:
+  push:
+  workflow_dispatch:
+  release:
+    types:
+      - published
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - id: var
+        name: Prepare environment
+        run: |
+          rm -rf ./build/*
+          mkdir ./build
```

Comment:
Is the `id` not obsolete here and is the `rm` really necessary? IIRC GH Actions have a clean workspace each time they run and since I don't see a `build` dir in the repo there *should* never be one at this point of the workflow.

Reply:
`id` is indeed a copypasta artifact.
The `rm` doesn't hurt, and I'd rather ensure the directory is present and empty, as we're attaching everything inside it to the GitHub releases.